portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,86 @@
+html {
+    background: url(images/try.gif);
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+
+h1 {
+    font-size: 100px;
+    color: white;
+    padding-top: 12%
+}
+
+h2 {
+    font-size: 50px;
+    color: white;
+    padding-top: 1.75%
+}
+
+p {
+    color: white;
+}
+
+h3 {
+    font-size: 30px;
+    color: white;
+    padding-top: 1%;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color:black; 
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
```

Comment:
Nice ordering here! I like that you're gong from less to more specific as you go down the file